src/test/scala/edu/ie3/simona/model/participant/PvModelITHelper.scala

```diff
@@ -0,0 +1,109 @@
+/*
+ * © 2020. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.model.participant
+
+import edu.ie3.simona.ontology.messages.services.WeatherMessage
+import edu.ie3.simona.test.common.input.PvInputTestData
+import edu.ie3.util.TimeUtil
+import edu.ie3.util.scala.quantities.WattsPerSquareMeter
+import org.apache.commons.csv.{CSVFormat, CSVRecord}
+import squants.energy.Megawatts
+import squants.motion.MetersPerSecond
+import squants.{Kelvin, Power}
+
+import java.io.{BufferedReader, File, FileInputStream, InputStreamReader}
+import java.nio.charset.StandardCharsets
+import java.time.ZonedDateTime
+import java.util.zip.GZIPInputStream
+import scala.jdk.CollectionConverters.IterableHasAsScala
+
+trait PvModelITHelper extends PvInputTestData {
+
+  private val CSV_FORMAT: CSVFormat =
+    CSVFormat.DEFAULT.builder().setHeader().get()
+
+  def getCsvRecords(fileName: String): Iterable[CSVRecord] = {
+    val resourcePath = getClass.getResource(fileName).getPath
+    val resultsInputData = new File(resourcePath)
+    val fileStream = new FileInputStream(resultsInputData)
+    val gzipStream = new GZIPInputStream(fileStream)
+    val decoder = new InputStreamReader(gzipStream, StandardCharsets.UTF_8)
+    val br = new BufferedReader(decoder)
+
+    CSV_FORMAT.parse(br).asScala
+  }
+
+  def createPvModels(): Map[String, PvModel] = {
+    pvInputsTest.map { inputModel =>
+      inputModel.getId -> PvModel(
+        inputModel,
+        1.0,
+        TimeUtil.withDefaults.toZonedDateTime("2011-01-01T00:00:00Z"),
+        TimeUtil.withDefaults.toZonedDateTime("2012-01-01T00:00:00Z"),
+      )
+    }.toMap
+  }
+
+  def getWeatherData
+      : Map[ZonedDateTime, Map[String, WeatherMessage.WeatherData]] = {
+    val fileName = "_pv/it/weather.tar.gz"
+    val csvRecords: Iterable[CSVRecord] = getCsvRecords(fileName)
+
+    csvRecords.foldLeft(
+      Map.empty[ZonedDateTime, Map[String, WeatherMessage.WeatherData]]
+    ) { (weatherDataMap, row) =>
+      val time = TimeUtil.withDefaults.toZonedDateTime(row.get(0))
+      val modelId = row.get(1)
+
+      val temp = 0.0
+      val windVel = 0.0
```

Comment:
A very minor thing... just to improve readability of the method call below a tiny bit I would suggest to use Kelven(0.0) here (same for wind speed).